libs/ledger-live-common/src/exchange/swap/types.ts

```diff
@@ -34,6 +34,35 @@ export type ExchangeRate = {
   expirationTime?: number;
 };
 
+type ExchangeRateV5ProviderType = "DEX" | "CEX" | "DISABLED";
```

Comment:
Are we receiving DISABLED type from backend?

Reply:
This was defined on the swagger file. But I've yet to see it.